Include/cpython/pyatomic_gcc.h

```diff
@@ -0,0 +1,679 @@
+#ifndef Py_ATOMIC_GCC_H
+#  error "this header file must not be included directly"
+#endif
+
+// This is the implementation of Python atomic operations using GCC's built-in
+// functions that match the C+11 memory model. This implementation is preferred
+// for GCC compatible compilers, such as Clang. These functions are available in
+// GCC 4.8+ without needing to compile with --std=c11 or --std=gnu11.
+
+static inline int
+_Py_atomic_add_int(int *address, int value)
+{
+    return __atomic_fetch_add(address, value, __ATOMIC_SEQ_CST);
+}
```

Comment:
Usually, I prefer verbose syntax like the one that you used. But in this header file, you have tons of static inline functions, so I suggest using the compact syntax:

```suggestion
static inline int
_Py_atomic_add_int(int *address, int value)
{ return __atomic_fetch_add(address, value, __ATOMIC_SEQ_CST); }
```